subprojects/language-java/src/main/java/org/gradle/api/internal/tasks/compile/incremental/recomp/GroovyRecompilationSpecProvider.java

```diff
@@ -126,7 +132,7 @@ private void processOtherChanges(PreviousCompilation previous, RecompilationSpec
             if (relativeSourceFile.isPresent()) {
                 spec.getRelativeSourcePathsToCompile().add(relativeSourceFile.get());
             } else {
-                spec.setFullRebuildCause("Can't find source file of class " + className, null);
+                spec.setFullRebuildCause("unable to find source file of class " + className, null);
```

Comment:
```suggestion
                spec.setFullRebuildCause("Unable to find source file of class " + className, null);
```

I think all the rebuild causes start with an uppercase letter.

Reply:
See my comment above for why it shouldn't be capitalized.